ovc5/software/libovc/include/libovc/server.hpp

```diff
@@ -78,7 +77,7 @@ class Server
 
   void stop();
 
-  std::array<OVCImage, NUM_IMAGERS> getFrames();
+  std::unordered_map<uint8_t, OVCImage> getFrames();
```

Comment:
nit, maybe a `map`? It might be nice to be able to iterate through the structure in a predictable way (increasing camera_id)